docs/api/security/platform_service.md

```diff
@@ -0,0 +1,11 @@
+## Platform service
+
+The Platform service introduces System Reset and PSA Lifecycle APIs.
```

Comment:
@alzix, can you please elaborate on what the platform service is? I see that it consists of the Lifecycle API, which we have documented separately, and the System Reset API. Since Lifecycle is already documented, should we just have a System Reset page? Is there anything else we can say about the system reset? When would you use it? Also, the class reference here is for the Lifecycle API. Can we put the System Reset API class reference instead?